.circleci/config.yml

```diff
@@ -40,7 +40,8 @@ commands:
           command: |
             . /opt/conda/etc/profile.d/conda.sh
             conda activate esmvaltool
-            pytest -n 4 --junitxml=test-reports/report.xml
+            mamba --version
+            pytest -n 4
```

Comment:
Why are you removing the bit that generates the test report for CircleCI?

Reply:
coz I am a doofus, I forgot this was used by Circle reporting :grin:

Reply:
put it back, sorry buds :beer: